documentation/STIR-UsersGuide.tex

```diff
@@ -3976,6 +3976,29 @@ \subsubsection{
 Otherwise, all non-positive entries in the data are set to SMALL\_NUM 
 times the minimum strictly positive value.
 
+{ \subsubsubsection{HUToMu}
+}
+
+This data processors is intended to convert a CT image in Hounsfield Units to mu-values.
+It takes a file in JSON format specifying the piecewise-linear for various systems, Xray voltages and gamma energies.
+
```

Comment:
Add:

"Currently, it supports 2-segment piecewise linear transformations only."

Reply:
(I am happy to add these myself, just noting them here for later)